contracts/uniswap/UniswapV2Router02.sol

```diff
@@ -0,0 +1,270 @@
+pragma solidity ^0.6.0;
+
+import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol";
+import "../interfaces/IERC20Extended.sol";
+import "../libraries/UniswapV2Library2.sol";
+import "./UniswapV2Router01.sol";
+
+contract UniswapV2Router02 is UniswapV2Router01 {
```

Comment:
this is a bit confusing.
in a uniswap repo I have
routerV2 doesn't inherit V1

its written from blank
maybe just take routerV2.

Reply:
Ok, change as request. Also add an diff checker to README.md